ts/features/itwallet/common/components/ItwUpgradeBanner.tsx

```diff
@@ -15,7 +15,8 @@ export const ItwUpgradeBanner = () => {
 
   const handleOnPress = () => {
     navigation.navigate(ITW_ROUTES.MAIN, {
-      screen: ITW_ROUTES.DISCOVERY.INFO
+      screen: ITW_ROUTES.DISCOVERY.INFO,
+      params: {}
```

Comment:
The upgrade banner should pass the `isL3` flag
```suggestion
      params: { isL3: true }
```